### DISCUSSION TITLE
docs/v1.12.x/guide/loadbalancer-service/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#2485] [Announcements]
# docs/v1.12.x/guide/loadbalancer-service/

https://kubeovn.github.io/docs/v1.12.x/guide/loadbalancer-service/

**Comment by hongzhen-ma** [comment]
annotation 中 attchmentprovider 拼写有误，应该为 attachmentprovider

**Reply by hongzhen-ma** [reply]
https://github.com/kubeovn/docs/pull/49

**Comment by Mrtao1028** [comment]
如何配置DNAT，用于获取请求的源IP
